OrbitCore/LinuxTracingHandler.cpp

```diff
@@ -16,6 +16,10 @@
 ABSL_FLAG(uint16_t, sampling_rate, 1000,
           "Frequency of callstack sampling in samples per second");
 
+// TODO: Remove this flag once we have an ui option to specify.
```

Comment:
please move this flag to main.cpp and then propagate it to this class - when this flag is set from UI it should not modify global but call a corresponding setter in this object.

Reply:
Well, how should this look like? Should I move the flag to OrbitGl/main.cpp? And then add a setter method for the unwinding method?

Reply:
Having the setter, would be then probably be multiple setter, right? Access to this instance of LinuxTracingHandler has currently OrbitAsioServer. So we would need to propagate that flag from Client to Service first.

Reply:
OrbitService main.cpp for now...

Reply:
Okay, I moved the flag accordingly.

Reply:
I don't know @dimitry- , now we have a `sampling_method` parameter everywhere and I can't say that I like it.

Reply:
it is in c-tor which is fine. And the services need a way to propagate it. I think this is fine, at least there is an explicit flag in LinuxTracingHandler which could now be set from UI.

Reply:
Then should we just propagate the whole `TracingOptions` struct?

Reply:
makes sense.

Reply:
Changed it, such that the whole `TracingOptions` will be passed.